src/tools/tidy/src/deps.rs

```diff
@@ -96,6 +96,7 @@ const WHITELIST: &[Crate<'_>] = &[
     Crate("lock_api"),
     Crate("log"),
     Crate("log_settings"),
+    Crate("measureme"),
```

Comment:
I don't know if there's an additional approval process for crate additions but I wanted to call this out just to be safe.

Reply:
I think that's OK since we already announced that we're going to add this crate.